sdk/search/Azure.Search/README.md

```diff
@@ -0,0 +1,171 @@
+# Azure Cognitive Search client library for .NET
+
+Azure Cognitive Search ([formerly known as "Azure Search"](https://docs.microsoft.com/azure/search/whats-new#new-service-name))
+is a search-as-a-service cloud solution that gives developers APIs and tools
+for adding a rich search experience over private, heterogeneous content in web,
+mobile, and enterprise applications. Your code or a tool invokes data ingestion
+(indexing) to create and load an index. Optionally, you can add cognitive
+skills to apply AI processes during indexing. Doing so can add new information
+and structures useful for search and other scenarios.
+
+On the other side of your service, your application code issues query requests
+and handles responses.  The search experience is defined in your client using
+functionality from Azure Cognitive Search, with query execution over a
+persisted index that you create, own, and store in your service.
+
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Cognitive Search client library for .NET with [NuGet][nuget]:
+
+```Powershell
+dotnet add package Azure.Search --version 1.0.0-preview.1
+```
+
+### Prerequisites
+
+You need an [Azure subscription][azure_sub] and a
+[Search Service][create_search_service_docs] to use this package.
+
+To create a new Search Service, you can use the [Azure Portal][create_search_service_docs],
+[Azure PowerShell][create_search_service_ps], or the [Azure CLI][create_search_service_cli].
+Here's an example using the Azure CLI to create a free instance for getting started:
+
+```Powershell
+az search service create --name mysearch --resource-group mysearch-rg --sku free --location westus
+```
+
+## Key concepts
+
+Azure Cognitive Search is the only cloud search service with built-in AI
+capabilities that enrich all types of information to easily identify and
+explore relevant content at scale.
+
+- Fully managed search as a service to reduce complexity and scale easily
+- Auto-complete, geospatial search, filtering, and faceting capabilities for a rich user experience
+- Built-in AI capabilities including OCR, key phrase extraction, and named entity recognition to unlock insights
+- Flexible integration of custom models, classifiers, and rankers to fit your domain-specific needs
+
+`Azure.Search` offers two types of resources:
+
+- The _search service_ used via `SearchServiceClient` to create and manage search indexes
```

Comment:
This whole document is meant to be a placeholder, but yes, I started with that because we'll be adding them first.  Happy to let you have total control of all this wording.

Reply:
I did a pretty thorough review of the Java README -- that would be a good starting point.